src/api/storage/v0/storage.ts

```diff
@@ -97,6 +102,28 @@ export class Bucket {
     }
     return new File(this.storage, this, name);
   }
+
+  /**
+   * Register and start a bucket notification handler that will be called for all matching notification events on this bucket
+   *
+   * @param notificationType the notification type that should trigger the middleware, either 'write' or 'delete'
```

Comment:
if it is either write or delete, should we make this the arg type instead of string for better DX typing?